src/KiwiLogin/__test__/KiwiLogin.test.js

```diff
@@ -1,23 +1,68 @@
 // @flow
-
 import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import { PureKiwiLogin } from '../index';
 
 describe('KiwiLogin', () => {
-  /*
-  enzyme broken until new release https://github.com/airbnb/enzyme/pull/1513
-
-  const mockStore = configureStore();
-  const store = mockStore({});
-
-  const component = mount(
-    <CloseContext.Provider value={() => {}}>
-      <MemoryRouter initialEntries={[{ key: 'testKey' }]}>
-        <KiwiLogin store={store} />
-      </MemoryRouter>
-    </CloseContext.Provider>
-  );
-  */
+  const props = {
+    onLogin: () =>
+      new Promise(resolve => {
+        process.nextTick(() => resolve());
```

Comment:
Just curious, why not just `new Promise(resolve => resolve())`, wouldn't it end up basically at the same queue position? :slightly_smiling_face:  Idem for Line 51 below.

Reply:
There is no specific reason behind this, I just always do it like this..
and now when you asked I realized it doesnt make sense 😂
I think the cleanest approach would be `async () => {}`

Reply:
Haha, fair enough. In your defense, jest's docs also use `nextTick` thing. 

`async () => {}` is nice but isn't `onLogin` specifically a `Promise` as received by props? https://github.com/kiwicom/smart-faq/blob/6e8fe3ad41bf65ab01ab47611e51ac99cb69cdec/src/types.js

Reply:
Ofc, please ignore my question. It should be totally equivalent :sweat_smile: